doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+      ],
+      "reference": "http://localhost:5050/notify",
+      "duration": "P1M",
+      "notifyConditions": [
+          {
+              "type": "ONCHANGE",
+              "condValues": [
+                  "title",
+                  "pages",
+                  "price"
+              ]
+          }
+      ],
+      "throttling": "PT5S"
+  }
+  EOF
+  ```
+
+  There are three JSON special parameters when a susbscription is sent:
+  * `attributes`: Refers to the attributes that will be notified to `Cygnus`. If empty, all attributes will be notified to `Cygnus`. If not, only the attributes wrote here will be notified. In this case, if `Orion` receive a change for the attribute `price` in entity `Book1` all the attributes (`title`, `pages` and `price`) will be sent to `Cygnys`.
+  * `condValues`: List of attributes that `Book1` has. When a subscription is done to an entity their attributes must be sent to `Orion`
+  * `reference`: Destination of notifications. In this case we use `http://localhost:5050/notify` but you can configure it with your own destination. This parameter involve that our `Cygnus` must be configured as destination.
+
+  Once sent, you will receive something like that:
+  ```
+  {
+      "subscribeResponse": {
+          "duration": "P1M",
+          "subscriptionId": "your_subscription_id",
+          "throttling": "PT5S"
+      }
+  }  
+  ```
+  The value `subscriptionId` is very important. It's highly recommended to save it in case of you need to unsubscribe to the entity in the future.
+
+  If you receive another answer from `Orion` check your curl: `JSON Parse Error` is a typical error. In that case you will receive this message:
+  ```
+  {
+      "subscribeError": {
+          "errorCode": {
+              "code": "400",
+              "details": "JSON Parse Error",
+              "reasonPhrase": "Bad Request"
+          }
+      }
+  }   
+  ```
+
+3. Append some values in entity `Book1`.
+  ```
+  (curl localhost:1026/v1/updateContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+  {  
+      "contextElements": [
+          {
+              "type": "Book",
+              "isPattern": "false",
+              "id": "Book1",
+              "attributes": [
+                  {
+                      "name": "title",
+                      "type": "text",
+                      "value": "Game of Thrones"
+                  },
+                  {
+                      "name": "pages",
+                      "type": "integer",
+                      "value": "927"
+                  },
+                  {
+                      "name": "price",
+                      "type": "float",
+                      "value": "18.50"
+                  }
+              ]
+          }
+      ],
+      "updateAction": "APPEND"
+  }
+  EOF
+  ```
+  This action only store a value in the attributes, isn't a change for notify to `Cygnus`. You will receive:
```

Comment:
In fact, I would check this point. AFAIK, when an entity is created a notification is sent as well.